src/install/config/rabbitmq.yaml

```diff
@@ -0,0 +1,94 @@
+agentName: rabbitmq
+agentType: integration
+title: 'RabbitMQ monitoring integration'
+metaDescription: 'Install the RabbitMQ integration'
+introFilePath: 'src/install/rabbitmq/intro.mdx'
+appInfo: 
+  - optionType: dropdown1
+    label: ''
+    placeholder: 'Select your environment'
+    options: 
+      - value: 'linux'
+        displayName: 'Linux'
+        logo: 'linux'
+        recommendedGuided: true
+      - value: 'windows'
+        displayName: 'Windows'
+        logo: 'windows'
+        recommendedGuided: true
+      - value: 'ecs'
+        displayName: 'ECS'
+        logo: 'amazonECS'
+      - value: 'kubernetes'
+        displayName: 'Kubernetes'
+        logo: 'k8'
+      - value: 'ansible'
+        displayName: 'Ansible'
+        logo: 'linux'
+
+  - optionType: deployment
```

Comment:
<img width="1153" alt="Screenshot 2024-03-12 at 4 04 33 PM" src="https://github.com/newrelic/docs-website/assets/42678939/b7f8c3b0-6c82-42ce-858b-216789f01801">

Why is there only Tarball as an option here? Looking at the exisitng doc, I think this should include the normal options of Yum, apt, etc.

Reply:
Added!